addons/website/static/src/interactions/popup/no_backdrop_popup.js

```diff
@@ -0,0 +1,83 @@
+import { registry } from "@web/core/registry";
+import { isScrollableY } from "@web/core/utils/scrolling";
+import { Interaction } from "@website/core/interaction";
+
+export class NoBackdropPopup extends Interaction {
+    static selector = ".s_popup_no_backdrop";
+    // TODO: handle edit mode
+    // disabledInEditableMode = false;
+    dynamicContent = {
+        "_root": {
+            "t-on-shown.bs.modal": this.onModalNoBackdropShown,
+            "t-on-hide.bs.modal": this.onModalNoBackdropHide,
```

Comment:
suggestion
```js
            "t-on-hide.bs.modal": this.removeModalNoBackdropEvents,
```
and you can remove `onModalNoBrackdropHIde` if you want

Reply:
Done :+1: